examples/trials/NAS/README.md

```diff
@@ -0,0 +1,8 @@
+ **Run Neural Network Architecture Search in NNI**	
+ ===	
+ 
+Now we have an NAS example [NNI-NAS-Example](https://github.com/Crysple/NNI-NAS-Example) run in NNI using NAS interface from our contributors.	
+ 
+Thanks our lovely contributors. 	

```

Comment:
lol :)

Reply:
I copy most of the README from ENAS_EXAMPLE OTZ